### ISSUE TITLE
login: all users treated as guests (no login possible)

### ISSUE THREAD

**Issue by jmartinm** [inveniosoftware/invenio#3460]
There is a problem in this line https://github.com/inveniosoftware/invenio/blob/master/invenio/ext/login/legacy_user.py#L159

```
*** AttributeError: 'User' object has no attribute 'groups'
```

which causes the following line to execute https://github.com/inveniosoftware/invenio/blob/master/invenio/ext/login/legacy_user.py#L166

That means that currently it is not possible to login into the system.

**Comment by kaplun** [comment]
Could it be connected with Invenio assuming invenio-groups is always there?

**Comment by jmartinm** [comment]
Yes, that is the problem actually. On INSPIRE we do not need `invenio_groups` in principle so I did not have it in `PACKAGES`.

The code should not fail `invenio_groups` is not installed.

**Comment by jirikuncar** [comment]
- related to #3438

**Comment by jirikuncar** [comment]
Closed via #3438
